var/spack/repos/builtin/packages/py-mdanalysis/package.py

```diff
@@ -38,7 +39,10 @@ class PyMdanalysis(PythonPackage):
     depends_on('py-six@1.4.0:', type=('build', 'run'))
     depends_on('py-biopython@1.59:', type=('build', 'run'))
     depends_on('py-networkx@1.0:', type=('build', 'run'))
-    depends_on('py-griddataformats@0.3.2:', type=('build', 'run'))
+    depends_on('py-griddataformats@0.4:', type=('build', 'run'))
+    depends_on('py-gsd', type=('build', 'run'))
+    depends_on('py-mmtf-python', type=('build', 'run'))
+    depends_on('py-scikit-learn', type=('build', 'run'))
 
```

Comment:
Missing dependencies on joblib, scipy, matplotlib, and mock, which are all required, at least in the latest release.